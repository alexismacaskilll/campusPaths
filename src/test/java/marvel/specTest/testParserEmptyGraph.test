# Test the parser on a file that is empty

# Load a graph from empty tsv file
LoadGraph emptyGraph empty.tsv

# Print the heros in the graph
ListNodes emptyGraph