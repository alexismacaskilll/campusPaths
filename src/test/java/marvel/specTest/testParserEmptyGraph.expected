# Test the parser on a file that is empty

# Load a graph from empty tsv file
loaded graph emptyGraph

# Print the heros in the graph
emptyGraph contains: